README.md

```diff
@@ -1 +1,135 @@
-# auth-gateway
\ No newline at end of file
+<p align="center"><img src="images/logo.png" alt="auth-gateway Logo"></p>
+
+[![CI](https://github.com/cortexproject/auth-gateway/workflows/ci/badge.svg)](https://github.com/cortexproject/auth-gateway/actions)
+[![GoDoc](https://godoc.org/github.com/cortexproject/auth-gateway?status.svg)](https://godoc.org/github.com/cortexproject/auth-gateway)
+<a href="https://goreportcard.com/report/github.com/cortexproject/auth-gateway"><img src="https://goreportcard.com/badge/github.com/cortexproject/auth-gateway" alt="Go Report Card" /></a>
+<a href="https://cloud-native.slack.com/messages/cortex/"><img src="https://img.shields.io/badge/join%20slack-%23cortex-brightgreen.svg" alt="Slack" /></a>
+<!-- <a href="https://bestpractices.coreinfrastructure.org/projects/6681"><img src="https://bestpractices.coreinfrastructure.org/projects/6681/badge"></a> -->
```

Comment:
We had to sign in and add `auth-gateway` to this website. However, I was not sure which e-mail address I should use to sign in. I though maybe there is an e-mail dedicated to Cortex so I did not add the project and commented out this line.

Reply:
There is an email for cortex mantainers, I will send you that email, we can use that